### PR TITLE
Setup working EfCore migration model with docker

### PR DESCRIPTION
**Pull request by HilmerChristophAkaHuima**
- Add db properties to the models
- IConfiguration to startup for connectionstring _(because docker should use environment variables but EfCore needs a connectionstring to generate a migration)_
- generate migration _(Add-Migration Init)_
- add connection string to appsettings.Development _(so the added connectionstring is only default at dev environment)_
- changed the EnsureCreated to Migrate again _(this migrates the pending migrations at startup **this should be changed in the near future to an extern migrate command**)_

Used Resources:
https://docs.microsoft.com/en-us/archive/msdn-magazine/2019/may/data-points-ef-core-in-a-docker-containerized-app-part-2
https://docs.microsoft.com/de-de/aspnet/core/fundamentals/configuration/?view=aspnetcore-3.1

### CODE REVIEW

**Review by dominikbraun — DISMISSED** [review]

**Review by aligator — DISMISSED** [review]
LGTM

File: ItemCatalogService.gRPCService/Startup.cs [hunk 1]
```diff
@@ -14,14 +15,20 @@ namespace ItemCatalogService.gRPCService
...
         // This method gets called by the runtime. Use this method to add services to the container.
         // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddGrpc();
-            
-            var connectionString = Environment.GetEnvironmentVariable("DB_CONNECTION_STRING");
-            services.AddDbContext<ItemContext>(options => options.UseNpgsql(connectionString));
+
+            services.AddDbContext<ItemContext>(options => options.UseNpgsql(Configuration["ConnectionStrings:DevConnectionPlSql"]));
```
Comment (muhmuhhum) [inline ItemCatalogService.gRPCService/Startup.cs]:
```suggestion
            services.AddDbContext<ItemContext>(options => options.UseNpgsql(Configuration.GetConnectionString("DevConnectionPlSql")));
```

Comment (muhmuhhum) [reply ItemCatalogService.gRPCService/Startup.cs]: For this the Connections string in appsettings.Development has to look like 
`"ConnectionStrings": {
      "DevConnectionPlSql": connectionString
}`

Comment (muhmuhhum) [reply ItemCatalogService.gRPCService/Startup.cs:31]: Look here [https://stackoverflow.com/questions/44931613/how-to-correctly-store-connection-strings-in-environment-variables-for-retrieval](url)

**Review by muhmuhhum — APPROVED** [review]

**Pull request merged**